python/sglang/srt/layers/moe/utils.py

```diff
@@ -57,6 +55,7 @@ class MoeRunnerBackend(Enum):
     FLASHINFER_MXFP4 = "flashinfer_mxfp4"
     FLASHINFER_CUTEDSL = "flashinfer_cutedsl"
     CUTLASS = "cutlass"
+    CUTLASS_FP8 = "cutlass_fp8"
```

Comment:
I'm going to merge this PR. Could you use CUTLASS here? I don't think we need a new backend name here.

Reply:
Thanks, fixed